components/authz-service/server/v2/authz.go

```diff
@@ -211,19 +211,34 @@ func (s *authzServer) ValidateProjectAssignment(
 		return &api.ValidateProjectAssignmentResp{}, nil
```

Comment:
I bet the comment above this function will need an update.

A super invalid case feels like they passed a non-empty array and also the IsCreate flag. Can we throw an error in that case so other teams consuming this in the future don't misuse it by accident?

Reply:
Ah I think `InvalidCreateRequest` is covering this?